glusterd2/volume/volume-utils.go

```diff
@@ -199,3 +200,53 @@ func CreateVolumeInfoResp(v *Volinfo) *api.VolumeInfo {
 		SnapList:  v.SnapList,
 	}
 }
+
+// MountLocalBricks mounts bricks of auto provisioned volumes
+func MountLocalBricks() error {
+	volumes, e := GetVolumes()
+	if e != nil || volumes == nil {
```

Comment:
This should be 2 unique cases. You can have an error trying to get volumes, and you can get an empty list of volumes. The error is an error, the empty list is not.